Cap framerate at 120 fps instead of 62.
Skip startup movie.

--- a/TdEngine.ini
+++ b/TdEngine.ini
@@ -137,7 +137,7 @@
 [Engine.GameEngine]
 bSmoothFrameRate=TRUE
 MinSmoothedFrameRate=22
-MaxSmoothedFrameRate=62
+MaxSmoothedFrameRate=120
 
 [Engine.DemoRecDriver]
 AllowDownloads=True
@@ -944,7 +944,7 @@
 LoadMapLoadTime=29.0f

 [FullScreenMovie]
-StartupMovies=StartupMovie
+;StartupMovies=StartupMovie
 SkippableMovies=Attract_Movie
 SkippableMovies=scene_01
 SkippableMovies=scene_02
